src/7-to-8/major-changes/compatibility-mode.rst

```diff
@@ -106,149 +183,91 @@ You must change the configuration yourself. In this case:
+and Cylc 8 backward compatibility mode, you could do something like this
+in the bash script:
 
 .. code-block:: cylc
-   :caption: Part of a Cylc global configuration
 
-   [platforms]
-       [[supercomputer_A]]
-           hosts = localhost
-           job runner = slurm
-       [[supercomputer_B]]
-           hosts = tigger, wol, eeyore
-           batch system = pbs
+   [runtime]
+       [[foo]]
+           script = """
+               if [[ "${CYLC_VERSION:0:1}" == 7 ]]; then
+                   cylc hold "$CYLC_SUITE_NAME"
+               else
+                   cylc pause "$CYLC_WORKFLOW_ID"
+               fi
+           """
 
-And you have a **cylc 7** workflow runtime configuration:
+Note this logic (and the ``$CYLC_VERSION`` environment variable) is executed
+at runtime on the :term:`job host`.
+
+Alternatively, you could use :ref:`Jinja` like so:
 
 .. code-block:: cylc
-   :caption: Part of ``suite.rc``
 
    [runtime]
-       [[task1]]
-           [[[job]]]
-               batch system = slurm
-       [[task2]]
-           [[[remote]]]
-               hosts = eeyore
-           [[[job]]]
-               batch system = pbs
+       [[foo]]
+           {% if CYLC_VERSION is defined and CYLC_VERSION[0] == '8' %}
+               script = cylc pause "$CYLC_WORKFLOW_ID"
+           {% else %}
+               script = cylc hold "$CYLC_SUITE_NAME"
+           {% endif %}
 
+Note this logic (and the ``CYLC_VERSION`` Jinja2 variable) is executed locally
+prior to Cylc parsing the workflow configuration.
 
-Then, ``task1`` will be assigned platform
-``supercomputer_A`` because the specified host (implicitly ``localhost``)
-is in the list of hosts for ``supercomputer_A`` **and** the batch system is the same.
-Likewise, ``task2`` will run on ``supercomputer_B``.
 
-.. important::
+.. _compat.eg.custom_remote_install:
 
-   For simplicity, and because the ``host`` key is a special case (it can
-   match and host in ``[platform]hosts``) we only show these two config keys
-   here. In reality, **Cylc 8 compares the whole of**
-   ``[<task>][job]`` **and** ``[<task>][remote]``
-   **sections and all items must match to select a platform.**
+Custom remote installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. _major-changes-compatibility-caveats:
-
-Caveats
--------
-
-The following are some caveats to be aware of when using Cylc 8
-Compatibility Mode.
-
-.. warning::
+If you want to include certain files and directories in remote installation,
+use :cylc:conf:`flow.cylc[scheduler]install`. To ensure your workflow is still
+interoperable with Cylc 7, wrap it in a Jinja2 check like so:
 
-   Cylc 6 syntax deprecated by Cylc 7 is now obsolete. Run ``cylc validate``
-   *with Cylc 7* on your ``suite.rc`` to check for deprecation warnings and fix
-   those before validating with Cylc 8.
+.. code-block:: cylc
 
-.. warning::
+   {% if CYLC_VERSION is defined and CYLC_VERSION[0] == '8' %}
+   [scheduler]
+       install = my-dir/, my-file
+   {% endif %}
```

Comment:
I get the 7/8 interop angle because of the config changed but I think this would be better off in the installation section for a cleaner separation rather than dipping into install without explaining it fully here, then covering it again there.